Library/Homebrew/utils/pypi.rb

```diff
@@ -224,9 +227,18 @@ def self.update_python_resources!(formula, version: nil, package_name: nil, extr
         package_name = list_entry["package_name"]
         extra_packages = list_entry["extra_packages"]
         exclude_packages = list_entry["exclude_packages"]
+        dependencies = list_entry["dependencies"]
       end
     end
 
+    missing_dependencies = (dependencies || []).reject { |dependency| Formula[dependency].any_version_installed? }
+    if missing_dependencies.present?
+      unless install_dependencies
+        odie "Missing formulae needed to update \"#{formula.name}\" resources: #{missing_dependencies.join(", ")}"
+      end
+      missing_dependencies.each { |dependency| ensure_formula_installed!(dependency) }
```

Comment:
```suggestion
      missing_dependencies.each(&method(:ensure_formula_installed!))
```

If I remember the syntax correctly